src/commands/scanner/run.ts

```diff
@@ -47,32 +48,13 @@ export default class Run extends ScannerCommand {
 			description: messages.getMessage('flags.rulesetDescription'),
 			longDescription: messages.getMessage('flags.rulesetDescriptionLong')
 		}),
-		// TODO: After implementing this flag, unhide it.
-		rulename: flags.string({
-			char: 'n',
-			description: messages.getMessage('flags.rulenameDescription'),
-			// If you're specifying by name, it doesn't make sense to let you specify by any other means.
-			exclusive: ['category', 'ruleset', 'severity', 'exclude-rule'],
-			hidden: true
-		}),
 		engine: flags.array({
 			char: 'e',
 			description: messages.getMessage('flags.engineDescription'),
 			longDescription: messages.getMessage('flags.engineDescriptionLong'),
-			options: [ENGINE.ESLINT, ENGINE.ESLINT_LWC, ENGINE.ESLINT_TYPESCRIPT, ENGINE.PMD, ENGINE.RETIRE_JS]
+			options: [...AllowedEngineFilters]
```

Comment:
This is a great pattern, seems much more maintainable.

Reply:
Thanks Jeff. Though when I was testing this, I realized that the array values were never displayed before and they still don't get displayed. The values would only get displayed in the help message when the type is `flags.enum`, but this breaks some more logic related to value parsing. I've left things the way they were except for fetching the values from Constants.